modules/015-admission-policy-engine/docs/FAQ.md

```diff
@@ -2,7 +2,14 @@
 title: "The admission-policy-engine module: FAQ"
 ---
 
-## How to extend Pod Security Standards policies?
+## How do I configure alternative security policy management solutions?
+
+For DKP to work correctly, extended privileges are required to run and operate system component payloads. If you are using some alternative security policy management solution (e. g., Kyverno) instead of the admission-policy-engine module, you have to configure exceptions for the following namespaces:
+
+- `kube-system`;
```

Comment:
```suggestion
For DKP to work correctly, extended privileges are required to run and operate system component payloads. If you are using some alternative security policy management solution (e. g., Kyverno) instead of the admission-policy-engine module, you have to configure exceptions for the following namespaces:
- `kube-system`;
```